### PR TITLE
[WIPTEST] Login functionality of cockpit. 

### PR DESCRIPTION
**Pull request by jdemon519**
{{pytest: cfme/tests/containers/test_webconsole_node.py -v --use-provider cm-env1 }}

Test the cockpit single sign-on functionality.

### CODE REVIEW

File: cfme/tests/containers/test_webconsole_node.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    """ Verify single sign-on to cockpit from master
+    """
+    navigate_to(Node, 'All')
+    tb.select('List View')
+    list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    nodes_instances = [r for r in list_tbl.rows()]
+    nodes_instances[0].row_element.click()
+    tb.select('Open a new browser window with Cockpit for this '
+              'VM.  This requires that Cockpit is pre-configured on the VM.')
+    assert sel.title() == 'Cockpit'
```
Comment (mfalesni) [inline cfme/tests/containers/test_webconsole_node.py]: I reckon this will leave the window open, right? It will probably need to be closed afterwards.

Comment (jdemon519) [inline cfme/tests/containers/test_webconsole_node.py]: @mfalesni Right, exactly. At this point in time, it opens up a new browser window which requires a user to log in, and we want the automatic sign-on. BTW, what do you recommend for switching to a new browser window? Thanks.

Comment (mfalesni) [inline cfme/tests/containers/test_webconsole_node.py]: Actually, I haven't ever done multi-window selenium stuff, so we'll have to figure it out somehow ...

Comment (jdemon519) [inline cfme/tests/containers/test_webconsole_node.py]: @mfalesni I'll do some digging :)

Comment (mshriver) [inline cfme/tests/containers/test_webconsole_node.py]: We need this multi-tab or multi-window support for the HTML console as well. I have a branch where I've prototyped trying to switch tabs in the browser module, but there was a bug preventing the tab from being selected. IIRC @jkrocil addressed this bug but I haven't circled back to the task yet. We'll likely include the task in an upcoming sprint, @jdemon519 I'll connect with you before I start working on it.

File: cfme/tests/containers/test_webconsole_node.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@pytest.mark.meta(blockers=[BZ(1406772)])
+def test_webconsole_node(provider):
+    """ Verify single sign-on to cockpit from master
+    """
+    navigate_to(Node, 'All')
+    tb.select('List View')
+    list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    nodes_instances = [r for r in list_tbl.rows()]
+    nodes_instances[0].row_element.click()
+    tb.select('Open a new browser window with Cockpit for this '
```
Comment (mshriver) [inline cfme/tests/containers/test_webconsole_node.py]: In 5.6.4 on PRT, this fails, did you mean to uncollect for `<=` '5.6'?  It looks like the button just isn't in the toolbar.

File: cfme/tests/containers/test_webconsole_node.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+              'VM.  This requires that Cockpit is pre-configured on the VM.')
+    port_num = ':9090/system'
+    url_cockpit = provider.hostname + port_num
+    sel.get(url_cockpit)
+    assert sel.title() == 'Cockpit'
```
Comment (mshriver) [inline cfme/tests/containers/test_webconsole_node.py]: This fails in PRT in 5.7.1:
>       assert sel.title() == 'Cockpit'
E       assert 'cmPRT-Master...os.******.com' == 'Cockpit'
E         - cmPRT-Master-01.rhq.lab.eng.bos.******.com
E         + Cockpit

cfme/tests/containers/test_webconsole_node.py:36: AssertionError

**Pull request closed without merging**